lib/splash.cjs

```diff
@@ -0,0 +1,57 @@
+const { remove, mkdirp } = require('fs-extra')
+const { join } = require('pathe')
+const sharp = require('sharp')
+
+const devices =  [
```

Comment:
You can remove this array, just import the `src/devices.ts` in `src/icon.ts`, and pass it to the splash screen fork as argvs, then you'll be able to have them available in the `splashCreation` function